domains/_vercel.lumi.json

```diff
@@ -4,6 +4,6 @@
     "discord": "563697359423406082"
   },
   "record": {
-    "TXT": "vc-domain-verify=lumi.is-a.dev,38de86570893517719b8"
+    "A": [76.76.21.21]
```

Comment:
this is incorrect. If you want to add the vercel IP, do that on `lumi.json` instead